project.mml

```diff
@@ -285,14 +285,22 @@ Layer:
       <<: *osm2pgsql
       table: |-
         (SELECT
```

Comment:
Does wrapping this in a superquery accomplish anything at all? It seems to me that leaving lines 287-290 and 302 out would yield exactly the same result?

Reply:
No, since then the query would return all polygons regardless of their `natural` column.